vendor/DeviceOrientationControls.js

```diff
@@ -85,13 +85,13 @@ THREE.DeviceOrientationControls = function ( object ) {
 
     if ( device ) {
 
-      var alpha = device.alpha ? THREE.Math.degToRad( device.alpha ) + scope.alphaOffset : 0; // Z
+      var alpha = device.alpha ? THREE.MathUtils.degToRad( device.alpha ) + scope.alphaOffset : 0; // Z
```

Comment:
Probably these Math/MathUtils can go in a separate PR? Unrelated to the AR cursor

Reply:
hmm these were in another PR that got merged they should disappear I will rebase

Reply:
ok cool thanks

Reply:
If we rebase this is pretty close to merge

Reply:
I'm still working on the new tick method for mousemove support for xr and entity rayOrigins